website/docs/r/data_protection_backup_policy_postgresql.html.markdown

```diff
@@ -0,0 +1,154 @@
+---
+subcategory: "DataProtection"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_protection_backup_policy_postgresql"
+description: |-
+  Manages a Backup Policy Postgre Sql.
+---
+
+# azurerm_data_protection_backup_policy_postgresql
+
+Manages a Backup Policy Postgre Sql.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "rg" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_data_protection_backup_vault" "example" {
+  name                = "example-backup-vault"
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
+  datastore_type      = "VaultStore"
+  redundancy          = "LocallyRedundant"
+}
+
+resource "azurerm_data_protection_backup_policy_postgresql" "example" {
+  name                = "example-backup-policy"
+  resource_group_name = azurerm_resource_group.rg.name
+  vault_name          = azurerm_data_protection_backup_vault.example.name
```

Comment:
should we merge this and just take the `azurerm_data_protection_backup_vault.example.id` value and pull the rest out?

Reply:
I think it's not necessary, it only reduces one argument.